apps/hexagon_api/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ ExternalProject_Add(x86_tvm_runtime_rpc
     "-DUSE_LIBBACKTRACE=OFF"
```

Comment:
I didn't catch the motivation on introduction of `hexagon_api` earlier this year, but to add to the backlog it would be nice if the `hexagon_api` was built automatically as part of a USE_HEXAGON=ON build. It's easy for users to forget to recompile the hexagon_api after making changes and only rebuilding tvm.